src/test/scala/com/databricks/spark/avro/AvroWriteBenchmark.scala

```diff
@@ -0,0 +1,73 @@
+package com.databricks.spark.avro
+
+import java.util.concurrent.TimeUnit
+
+import scala.collection.JavaConversions._
+import scala.util.Random
+
+import com.google.common.io.Files
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql._
+import org.apache.spark.sql.test.TestSQLContext
+import org.apache.spark.sql.test.TestSQLContext._
+
+
+/**
+ * This object runs a simple benchmark test to find out how long does it take to write a large
+ * SchemaRDD to an avro file. It reads one argument, which specifies how many rows does the
+ * SchemaRDD that we're writing contain.
+ */
+object AvroWriteBenchmark {
+
+  val defaultNumberOfRows = 50000
```

Comment:
Oh, it should not throw an OOM based on size, that just made me realize we're not generating the rows sanitarily. We should be generating them like

```scala
sc.parallelize(0 until numberOfRows).map(_ => generateRandomRow())
```

(really we shouldn't even be parallelizing that range, but that should be fine for numberOfRows < a billion)